src/engine/Settings.cs

```diff
@@ -2,14 +2,21 @@
 using Newtonsoft.Json;
 
 /// <summary>
-///   Main object for containing player changeable game settings
+///   Class that handles storing and applying player changeable game settings.
 /// </summary>
 public class Settings
 {
-    private static readonly Settings SingletonInstance = LoadSettings();
+    // Singleton used for holding the live copy of game settings.
+    private static readonly Settings SingletonInstance = new Settings();
 
     static Settings()
     {
+        if (!SingletonInstance.LoadFromFile())
+        {
+            // Loading failed so we'll load the new defaults and save to create the configuration file.
+            SingletonInstance.LoadDefaults();
+            SingletonInstance.Save();
+        }
```

Comment:
I don't like this change. I much preferred that the readonly field initialization performed the settings creation, as I think this static block runs when the C# binary is loaded, meaning it runs too early for my liking.

Reply:
I can change it back, I think you're right the static constructor runs early on vs the field initializer only being run when it is accessed. I could just make a static method like before if you want, that always returns some kinda settings object.

Reply:
>  I could just make a static method like before if you want, that always returns some kinda settings object.

Yeah. That was the way it was before, right?

Reply:
Basically yes.